policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/reduction/three_children.yaml

```diff
@@ -0,0 +1,26 @@
+description: Manitoba Child Benefit phase out rate for families with three children.
```

Comment:
Manitoba phases the child benefit out for families with three children at this rate.